openquake/calculators/postproc/compute_rtgm.py

```diff
@@ -85,12 +85,17 @@
 # NOTE: for meanHCs_afe_RTGM and disaggr_by_src we want to display these
 # three imts, that are mandatory in this context. For the plot of governing
 # MCE we read imts from the imtls
-IMTS = ['PGA', 'SA(0.2)', 'SA(1.0)']
+IMTS = ['PGA', 'SA(0.02)', 'SA(0.03)', 'SA(0.05)', 'SA(0.075)', 'SA(0.1)',
+        'SA(0.15)', 'SA(0.2)', 'SA(0.25)', 'SA(0.3)', 'SA(0.4)', 'SA(0.5)',
+        'SA(0.75)', 'SA(1.0)', 'SA(1.5)', 'SA(2.0)', 'SA(3.0)', 'SA(4.0)',
+        'SA(5.0)', 'SA(7.5)', 'SA(10.0)']
```

Comment:
I thought most GMPEs could not be extended to SA(10.0) and we decided to discard it (?)

Reply:
We have now updated so that the IMTS are defined in the job file, instead of using defaults